docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,610 +1,127 @@
-
-### Update GPO to use a new certificate template
-
-In the group policy editor for `Teleport Access Policy`, select:
-
-```
-Computer Configuration > Policies > Administrative Templates > Windows Components > Remote Desktop Services > Remote Desktop Session Host > Security
-```
-
-Right-click on `Server authentication certificate template`, `Edit`, then select `Enabled` and fill `Certificate Template Name` with `RemoteDesktopAccess`.
-
-<Figure align="left" bordered caption="RDP Certificate Template">
-  ![RDP Certificate Template](../../img/desktop-access/rdp-certificate-template.png)
-</Figure>
-
-
-### Configure server certificate auto-enrollment
-
-In the group policy editor for `Teleport Access Policy`, select:
-
-```
-Computer Configuration > Policies > Windows Settings > Public Key Policies
-```
+</Details>
 
-Double-click on `Certificate Services Client - Auto-Enrollment`, then select `Enabled` in the `Configuration Model`.
+This guide will help you configure Teleport and your Active Directory domain controller to communicate securely with each other, register available desktops with your Teleport cluster, and provide secure access to them.
 
+## Prerequisites
 
-### Ensure your GPO is updated
+<ScopedBlock scope={["oss", "enterprise"]}>
 
-If you have not done so already, ensure your GPO is updated by opening a PowerShell prompt and running:
+- One or more hosts to run the Teleport Auth and Proxy services on.
 
-```powershell
-gpupdate.exe /force
-```
+</ScopedBlock>
+- A server or virtual machine running a Windows Server operating system.
 
-## Step 5/7. Export your LDAP CA certificate
+- A host where you will run the Teleport Desktop Service.
+  <ScopedBlock scope={["oss", "enterprise"]}> This **can not** be the same instance
+  of Teleport running your Auth and/or Proxy service(s). To install Desktop Access into an existing Teleport instance running other services, see the [Manual Setup](manual-setup.mdx) guide for Desktop Access.</ScopedBlock>
 
-Teleport connects to your Domain Controller via LDAPS. This means that you must
-let Teleport know that the certificate sent by your Domain Controller during the
-initial SSL connection is trusted. If your Domain Controller's certificate is
-trusted by the system repository on the system running Teleport, you can skip
-this step.
+  See [Installation](../installation.mdx) for details.
 
-<Details title="Skipping TLS Verification">
-  If you are unable to acquire the LDAP CA certificate, you can skip
-  TLS verification by setting `insecure_skip_verify: true`. We do not recommend
-  skipping TLS verification in production environments.
-</Details>
+## Step 1/2. Install Teleport
 
-### To export a CA certificate
+### Set up the Teleport Auth and Proxy Services
 
-{/* Adapted from https://www.ibm.com/docs/it/rds/5.2.1?topic=security-exporting-certificate-from-active-directory-server */}
+(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
```

Comment:
Shouldn't this be under the oss/enterprise scoped block?

Reply:
The partial file itself has scoped tabs:

```
<Tabs> 
<TabItem scope={["oss","enterprise"]} label="Self-Hosted">
...
</TabItem> 
<TabItem label="Teleport Cloud" scope={["cloud"]}> 
...
</TabItem>
</Tabs>

![image](https://user-images.githubusercontent.com/2349184/201378019-9102a5be-029a-4d9d-a64d-4402674c5d98.png)